src/components/Graph/Graph.tsx

```diff
@@ -137,15 +138,24 @@ const Graph: FC<GraphProps> = ({
             width: 400
           };
         } else {
+          // TODO: This obviously has to be changed eventually.
+          // The nodes height should change dynamically
+          const found =
+            evaluatedPathway &&
+            evaluatedPathway.pathwayResults &&
+            evaluatedPathway.pathwayResults.documentation.find(doc => {
+              return typeof doc !== 'string' && doc.state === e;
```

Comment:
As discussed offline -- is there anything we can do here to ensure the node height is always at least large enough to accommodate the contents? Doesn't have to be perfect at this point so if it overshoots that's fine but better to have too much space than overlapping nodes.